internal/linker/linked_example.go

```diff
@@ -66,13 +65,20 @@ func (e *LinkedExample) Dependencies() []string {
 		deps = append(deps, child.Name)
 	}
 
+	return append(e.SetupDependencies(), deps...)
+}
+
+// SetupDependencies returns setup dependecies for this example
+func (e *LinkedExample) SetupDependencies() []string {
```

Comment:
SetupDependencies confuses a bit. Consider renaming this.

Reply:
Done